src/main/java/com/hubspot/jinjava/lib/filter/time/DateTimeFormatHelper.java

```diff
@@ -0,0 +1,87 @@
+package com.hubspot.jinjava.lib.filter.time;
+
+import com.hubspot.jinjava.interpret.InvalidArgumentException;
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+import com.hubspot.jinjava.lib.fn.Functions;
+import com.hubspot.jinjava.objects.date.InvalidDateFormatException;
+import java.time.DateTimeException;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.IllformedLocaleException;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.function.Function;
+
+final class DateTimeFormatHelper {
+  private final String name;
+  private final Function<FormatStyle, DateTimeFormatter> cannedFormatterFunction;
+
+  DateTimeFormatHelper(
```

Comment:
Personally not a fan of implied access control but curious to hear your perspective

Reply:
I wanted this class to be an implementation detail and exclude it from the public API of Jinjava to avoid third parties relying on it. I think that makes package-private (not using an access modifier) the right tool for the job here.

Technically, as long as the class is package-private and `final` (which it is), I could make the constructor and methods `public` and they still can't be accessed by anything outside the package. But I think sticking with package-private helps communicate that this class is only meant to be used in this package.

Does that make sense to you?